### PR TITLE
use bootstrap-loader with bootstrap-sass

### PR DESCRIPTION
**Pull request by damirv**
Possible fix for #474.

@davezuko Not sure if this is what you had in mind, but it worked for me :smile:

### CODE REVIEW

**Comment by davezuko** [comment]
Hey, thanks for the PR. I'm super swamped with work right now and haven't had much of a chance to sit down and sift through issues/PR's. Will get to this as soon as things settle down a bit.

**Comment by damirv** [comment]
Sure, no problem :)

**Comment by davezuko** [comment]
Just curious, what's the benefit of using bootstrap-loader over a regular import in, say, `core.scss`? Not at all saying it's wrong, just would not have considered that approach.

**Comment by damirv** [comment]
I couldn't get it to work by placing the imports in `core.scss`, I get the following error:
```
ERROR in ./~/css-loader?modules&sourceMap&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/postcss-loader!./~/sass-loader?sourceMap!./src/styles/core.scss
Module build failed: CssSyntaxError: postcss-modules-local-by-default: /Users/damirv/Development/dev/src/styles/core.scss:1:0: Missing whitespace before :global
:global {
```

What i also find weird is the path: `/Users/damirv/Development/dev/src/styles/core.scss`, the actual path is `/Users/damirv/Development/dev/react-redux-starter-kit/src/styles/core.scss` (note the `react-redux-starter-kit` after `/dev/`).

Anyways, the [bootstrap-loader](https://github.com/shakacode/bootstrap-loader) package gives you control of bootstrap styling / component selection through the use of a config file (.bootstraprc), for some this might be an advantage but I can imagine that others want to have manual control. Let me know what your thoughts are on this.

File: package.json [hunk 1]
```diff
@@ -64,10 +64,13 @@
   "author": "David Zukowski <david@zuko.me> (http://zuko.me)",
   "license": "MIT",
   "dependencies": {
+    "bootstrap-loader": "^1.0.8",
+    "bootstrap-sass": "^3.3.6",
     "co-request": "^1.0.0",
     "debug": "^2.2.0",
     "history": "^2.0.0",
     "iconv-lite": "^0.4.13",
+    "jquery": "^2.2.0",
```
Comment (davezuko) [inline package.json]: I really want to pull this in, but this makes me so sad :(

**Comment by Grobim** [comment]
Hi,

I just tried this solution but my app won't start :

bootstrap.js:8 Uncaught Error: Bootstrap's JavaScript requires jQuery
transition.js:59  Uncaught ReferenceError: jQuery is not defined

(Win7, npm : 3.5.3)
(I cloned @damirv repo directly and runned : npm install && npm start)

**Comment by kevinchiu** [comment]
I'm currently using Bootstrap with this kit without any problems. I just put the bootstrap distributable files into src/static/bootstrap/ and added them as normal css/js files in index.html.\:

`
  <script src='/jquery-2.2.0.min.js'></script>
  <script src='/bootstrap/js/bootstrap.min.js'></script>
  <link rel='stylesheet' type='text/css' href='/bootstrap/css/bootstrap.min.css'>
`

**Comment by davezuko** [comment]
Fairly sure I'll just be removing bootstrap from this project altogether and replacing it with a recipe (as well as recipes for `react-toolbox`, `react-materials`, and maybe `semantic`). Really apologize for allowing this PR to linger for so long, I really just wanted to avoid the jQuery dependency and also making this a bootstrap-based project, since it really shouldn't be.

Moving to recipes, thanks to all of you for your contributions and patience.

**Comment by gomezfx** [comment]
@tgfisher Late reply, but maybe this will help someone else. Yeah, using bootstrap-loader, you can setup a .bootstraprc configuration file and then point to a sass file with your customizations.

**Comment by o1dnik** [comment]
@kevinchiu it doesn't solve customization issue with `bootstrap-sass`
@gomezfx but less third-party dependencies makes the project better. I would like to slice and customize in the way I want w/o dependency on bloated `bootstrap`.

File: package.json (hunk 1 above)
Comment (slamdev) [inline package.json]: what about replacing it by https://github.com/thednp/bootstrap.native

**Pull request closed without merging**